articles/virtual-machines/automatic-vm-guest-patching.md

```diff
@@ -214,12 +214,18 @@ PUT on `/subscriptions/subscription_id/resourceGroups/myResourceGroup/providers/
 }
 ```
 
-### Azure PowerShell for Windows VMs
-Use the [Set-AzVMOperatingSystem](/powershell/module/az.compute/set-azvmoperatingsystem) cmdlet to enable automatic VM guest patching when creating or updating a VM.
+### Azure PowerShell when creating Windows VMs
+Use the [Set-AzVMOperatingSystem](/powershell/module/az.compute/set-azvmoperatingsystem) cmdlet to enable automatic VM guest patching when creating a VM.
 
 ```azurepowershell-interactive
 Set-AzVMOperatingSystem -VM $VirtualMachine -Windows -ComputerName $ComputerName -Credential $Credential -ProvisionVMAgent -EnableAutoUpdate -PatchMode "AutomaticByPlatform"
 ```
+### Azure PowerShell when updating Windows VMs
+Use the [Update-AzVMOperatingSystem](/powershell/module/az.compute//update-azvm) cmdlet to enable automatic VM guest patching when updating an existing VM.
+
+```azurepowershell-interactive
+Update-AzVMOperatingSystem -VM $VirtualMachine -Windows -ComputerName $ComputerName -Credential $Credential -ProvisionVMAgent -EnableAutoUpdate -PatchMode "AutomaticByPlatform"
```

Comment:
`Update-AzVMOperatingSystem` does not exist. In both cases, the user must use Set-AzVMOperatingSystem to set the value on a PSVirtualMachine and then either New-AzVM or Update-AzVM to invoke the underlying REST API.

In this case, it would be something like:
$virtualMachine = Get-AzVM <params...>
$virtualMachine = Set-AzVMOperatingSystem -VM $virtualMachine -patchMode AutomaticByPlatform
Update-AzVM -VM $virtualMachine